backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -74,6 +76,7 @@ export const executeTransaction = async (
   )
   logger.debug(`calculated Balance=${sendBalance}`)
   if (!sendBalance) {
+    // josejgi: wrong messages from my point of view, at this point balance is always null, should log inside calculate balance or handle returns in a different way
```

Comment:
sendBalance is null if there is no transaction present (means that the balance of the user is 0), or the balance is less than the amount to send. I agree that the return value should be improved in the case of having not enough GDD

Reply:
and notice that using sendBalance inside the log message is useless, as it will be null,
but as mentioned many times, I think that checkBalance() is a mess